### PR TITLE
[Tests-Only] Wait for the share notifications message to be displayed properly

### PR DESCRIPTION
**Pull request by HariBhandari07**
## Description

This scenario step 
```
    Then the user should see 2 notifications on the webUI with these details
      | title                                                                 |
      | "Carol King" shared "simple-folder" with you |
      | "Carol King" shared "data.zip" with you         |
```
fails intermittently when we are trying to read the notification message. Tests were failing because we were trying to read the notification title before it is displayed correctly on the webUI. This PR fixes that.
## Related Issue 

- Fixes https://github.com/owncloud/web/issues/4828

## How Has This Been Tested?

- manual
- ci

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingNotifications</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13771/

<p>








</p></details>

File: tests/acceptance/pageObjects/webPage.js [hunk 1]
```diff
@@ -81,13 +81,16 @@ module.exports = {
     getNotifications: async function() {
       const notifications = []
       await this.toggleNotificationDrawer()
-      await this.api.elements('@notificationElement', result => {
-        for (const element of result.value) {
-          this.api.elementIdText(element.ELEMENT, text => {
-            notifications.push(text.value)
-          })
+      await this.waitForElementVisible('@notificationElement').api.elements(
```
Comment (HariBhandari07) [inline tests/acceptance/pageObjects/webPage.js:84]: This is the only change that is required for the fix. All the other changes made in this PR (drone and feature file) will be removed after the approval/review.

**Review by dpakach — COMMENTED** [review]
lgtm :+1:

**Comment by jasson99** [comment]
LGTM. May be we can revert the other changes now

**Review by jasson99 — COMMENTED** [review]
LGTM. May be we can revert the other changes now

**Pull request closed without merging**